cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/VirtualOutletController.java

```diff
@@ -44,9 +47,7 @@ public List<VirtualOutlet> retrieveVirtualOutlets(ApiConnectionInfo apiConnectio
     public void storeVirtualOutlet(ApiConnectionInfo apiConnectionInfo, VirtualOutletEndpointInput.Post input)
             throws IOException {
         String body = RadarObjectMapper.mapObjectToJson(input.outletName());
-        new HttpRequestBuilderImpl(apiConnectionInfo, ENDPOINT_PREFIX
-                + input.outletName().getVirtualOutletId().getName() + "/"
-                + input.outletName().getVirtualOutletId().getOfficeId())
+        new HttpRequestBuilderImpl(apiConnectionInfo, ENDPOINT_PREFIX + MIDPOINT)
```

Comment:
You end up with `projects//virtual-outlets` here, I'd say remove the /'s from the endpoint and midpoint constants, and add them manually where necessary.

Reply:
string format method is probably the easiest way to make sure it's right. we probably should have been doing that everywhere...

Reply:
Replaced with manual "/" in latest push

Reply:
I would be inclined to do that, but after looking up performance comparisons, String.format is significantly slower than String concatenation.  From what I could tell, we're talking about 10x slower.  I think at this layer, it's best to be as performant as we can be.

Reply:
I bet we could make a utility method for constructing these paths, given a varargs of strings.  i.e. buildPath(String ... args), that just joins the text using /'s.  That might make this a bit more readable, but I don't know if it would be similarly performant.

Reply:
readability is more important than performance at this layer given the HTTP connection will be many orders of magnitude slower and we're not doing this in an iteration over data. The other performance issue is taking into consideration is the string pool, where we'd maybe want to consider using the StringBuilder. 

All that said, we're dealing with small strings and small concatenation so it isn't worth getting too hung up about.